src/lib/bootloader/udev_mapping.rb

```diff
@@ -38,26 +38,15 @@ def to_kernel_device(dev)
 
     # Converts udev or kernel device (disk or partition) to udev name that fits best.
     #
-    # Here is description of strategy for finding the best possible udev persistent name.
     # There are three scenarios we consider:
     # S1. disk with boot configuration is moved to different PC
     # S2. disk dies and its content is loaded to new disk from backup
     # S3. path to disk dies and disk is moved to different one
```

Comment:
I think this part of comment is also not needed as it is related to strategy. It is scenarios we try to support.

Reply:
Yes, I was not sure about removing it or not, because somehow it documents what scenarios bootloader expects to support. But I agree that is something that does not depend on bootloader anymore. I will drop it out.